docs/RELEASE.md

```diff
@@ -4,11 +4,16 @@ To release a new version of Kuberhealthy:
 
 1. Ensure that all issues in the release milestone are completed
 1. Verify that the master branch has all the latest changes that have been tested and verified
-1. After any running `Build and Push kuberhealthy Latest` action is completed, test the image `kuberhealthy/kuberhealthy:latest` to validate the release
-1. If the release looks good, tag it with the new upcoming version and push it to docker hub:
+1. After any running `Build and Push kuberhealthy Unstable` action is completed, tag a release candidate from the build done in master as unstable
     ```
-    docker pull kuberhealthy/kuberhealthy:latest
-    docker tag kuberhealthy/kuberhealthy:latest kuberhealthy/kuberhealthy:v[version]
+    docker pull kuberhealthy/kuberhealthy:unstable
+    docker tag kuberhealthy/kuberhealthy:v[version]-rc1 kuberhealthy/kuberhealthy:v[version]
```

Comment:
i believe `rc1` should be in the form `rc.1` -- https://semver.org/#spec-item-9

Reply:
👍